starknet_py/tests/e2e/conftest.py

```diff
@@ -188,7 +213,12 @@ async def address_and_private_key(pytestconfig, network, gateway_client):
 
 
 @pytest.fixture(scope="module")
-def gateway_account_client(address_and_private_key, gateway_client):
+def gateway_account_client(
+    address_and_private_key: Tuple[str, str], gateway_client: GatewayClient
+) -> AccountClient:
+    """
+    Returns an AccountClient with GatewayClient inside
```

Comment:
```suggestion
    Returns an AccountClient created with GatewayClient
```
maybe it sounds a little bit better?